crates/iota-graphql-client/src/lib.rs

```diff
@@ -2319,9 +2319,9 @@ mod tests {
             .await
             .unwrap()
             .unwrap();
-        assert_eq!(
-            total_transaction_blocks,
-            total_transaction_blocks_by_seq_num
+        assert!(
+            total_transaction_blocks_by_seq_num >= total_transaction_blocks,
+            "expected at least {total_transaction_blocks} transaction blocks, found {total_transaction_blocks_by_seq_num}"
```

Comment:
this is essentially a race condition, no? Wondering whether we should just change the assert to also being just positive. :thinking: